### PR TITLE
Reittiopas

### PR DESCRIPTION
**Pull request by przymi**
New Reittiopas app
Specify stop id and it will fetch the next buses or other means of transportation that will depart from there and display them.

### CODE REVIEW

File: apps/reittiopas.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+from gql import gql, Client
+from gql.transport.requests import RequestsHTTPTransport
+
+import time
+import logging
+import requests
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_STOP = "1432164" # HSL:1432164
```
Comment (jammers-ach) [inline apps/reittiopas.py]: maybe  ` os.getenv('DEFAULT_STOP', "1432164") `?

**Review by jammers-ach — APPROVED** [review]
Looks great!

File: apps/reittiopas.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
+from ti700.app import TerminalApp
+from gql import gql, Client
```
Comment (jammers-ach) [inline apps/reittiopas.py:2]: This needs to be added to `requirements.txt`

File: apps/reittiopas.py (hunk 1 above)
Comment (przymi) [reply apps/reittiopas.py]: Added

File: apps/reittiopas.py (hunk 2 above)
Comment (przymi) [reply apps/reittiopas.py:2]: Added

**Comment by przymi** [comment]
All requested changes made. Fell free to merge.

**Review by jammers-ach — APPROVED** [review]

**Pull request merged**